content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -12,6 +12,10 @@ weight: 10
 
 ## Security
 
+- **Added** the new [Authorization Policy](/blog/2019/v1beta1-authorization-policy/)
+- **Added** experimental support for [automatic mutual TLS](docs/tasks/security/authentication/auto-mtls/).
+- **Added** experimental [DNS Certificate Management](/blog/2019/dns-cert/)
+- **Added** support for [trust domain migration](/docs/tasks/security/authorization/authz-td-migration/)
```

Comment:
this one does not have experimental, wording, should it be? @myidpt @liminw

Reply:
Yes I think it should be listed as experimental. I mentioned it in the user guide, but we should have it here as well.